src/components/EditWalletDrawer/index.tsx

```diff
@@ -1,40 +1,112 @@
+import Modal from 'antd/lib/modal/Modal';
 import {inject, observer} from 'mobx-react';
 import * as React from 'react';
 import {WalletModel} from '../../models';
 import {RootStore} from '../../stores';
 import Drawer from '../Drawer';
 import EditWalletForm from '../EditWalletForm/index';
+import {Icon} from '../Icon/index';
+import {Link} from '../Link/index';
 import {Wizard, WizardStep} from '../Wizard';
 
 interface EditWalletDrawerProps {
   wallet?: WalletModel;
   show?: boolean;
+  toggleConfirmRemoveWallet?: () => void;
```

Comment:
i don't think `toggleConfirmRemoveWallet` and `showConfirmRemoveWallet` should be exposed in component api

Reply:
agreed, I will remove them